tooling/minimize/tests/ub/tree_borrows/overlapping_write_write.rs

```diff
@@ -0,0 +1,17 @@
+extern crate intrinsics;
+use intrinsics::*;
+
+//@ compile-flags: --minimize-tree-borrows
+fn main() {
+    let parent = &mut 31; // (parent, Reserved)   
+    let x = parent as *mut i32; // (parent, Reserved) 
+    let y = unsafe { &mut *x }; // (parent, Reserved) -> (y, Reserved)
+    *y = 42;
+
+    unsafe { 
+        *x = 57; // (parent, Reserved) -> (y, Disabled)
+        print(*x); // (parent, Reserved) -> (y, Disabled)
+    } 
+
+    *y = 31; // UB! Child Write to Disabled 
```

Comment:
Here this could in fact just do a read from y, even that should be UB. So add a third test for that, "overlapping_write_read".

Also, the tests are about mutable references. So maybe rename them to `mutable_ref_conflicting_read_write` etc?